packages/teleport/src/config.ts

```diff
@@ -268,9 +268,15 @@ const cfg = {
   },
 
   getPlayerRoute(params: UrlPlayerParams, search: UrlPlayerSearch) {
-    return `${generatePath(cfg.routes.player, { ...params })}?recordingType=${
-      search.recordingType
-    }`;
+    let route = `${generatePath(cfg.routes.player, {
+      ...params,
+    })}?recordingType=${search.recordingType}`;
```

Comment:
rant, prettier and its 80 col limit
```suggestion
   let route = generatePath(cfg.routes.player, {...params})
   route = `${route}?recordingType=${search.recordingType}`;
```